client-js/app/elements/labrad-manager.html

```diff
@@ -0,0 +1,68 @@
+<link rel="import" href="app-link.html">
+
+<dom-module id="labrad-manager">
+  <style>
+    :host {
+      display: block;
+    }
+  </style>
+  <template>
+    <table>
+      <tr>
+        <th>Id</th>
+        <th>Type</th>
+        <th>Name</th>
+        <th>Server Reqs</th>
+        <th>Server Reps</th>
+        <th>Client Reqs</th>
+        <th>Client Reps</th>
+        <th>Messages Sent</th>
+        <th>Messages Recv</th>
+      </tr>
+      <template is="dom-repeat" items="{{connections}}">
```

Comment:
I tried makign this comment, but it didn't seem to take, so my apologies if this comment shows up twice:

I would use ```
<thead>
  <th>Id</th>
 ...
</thead>
<tbody>
  <tr>
    <td>
    ...
</body
```

This gives separate CSS attributes we can mess with later when rendering the table, you can take a look at how I was rendering the registry table in the branch ```u/joshmutus/html```